### PR TITLE
Add change diff to RecordRequest endpoint

### PR DESCRIPTION
**Pull request by ghost**
`My Request` tabs needs information about changes making by request.
This change adds the information to `RecordRequest` endpoint.

### CODE REVIEW

File: powerdns/models/powerdns.py [hunk 1]
```diff
@@ -612,6 +620,28 @@ def can_auto_accept(self, user):
...
+            'content': '',
+            'name': '',
+            'owner': '',
+            'prio': '',
+            'remarks': '',
+            'ttl':  '',
+            'type':  '',
+        }
+
+    #TODO:: better names?
```
Comment (mkurek) [inline powerdns/models/powerdns.py]: 👍

File: powerdns/models/requests.py [hunk 2]
```diff
@@ -46,6 +47,7 @@ class Meta:
         null=True,
         blank=True
     )
+    last_change_json = models.TextField(null=True, blank=True)
```
Comment (mkurek) [inline powerdns/models/requests.py]: consider jsonfield for this (http://django-extensions.readthedocs.io/en/latest/field_extensions.html?highlight=json) - you're using django-extensions right now either way.

File: powerdns/models/powerdns.py (hunk 1 above)
Comment (ghost) [inline powerdns/models/powerdns.py]: any suggestion? :)

**Comment by szok** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**